packages/nx/src/utils/package-manager.ts

```diff
@@ -52,6 +59,8 @@ export function getPackageManagerCommand(
       exec: 'yarn',
       run: (script: string, args: string) => `yarn ${script} ${args}`,
       list: 'yarn list',
+      pack: 'npm pack',
+      view: 'npm view',
```

Comment:
Should this be `yarn pack` and `yarn view`?

Reply:
I did it because @leosvelperez's code overridden `yarn` with `npm` for these commands, I assumed that he'd done it for a reason.

I'm looking into it now, what difference is there between `yarn view/pack` and `npm view/pack`.

The least I can say is that `yarn` does not have `view` or equivalent command (from my 5 mins of research).
I also see that `yarn pack` does return the output file path, so it requires more work to understand the output file name, compared to `npm pack` which prints the output file path to `stdout` (and a whole bunch more to `stderr`).

Reply:
Yarn has `info` as a similar command to `view`, but it has differences that make it non-reliable:

```bash
npm view nx@14.0.0 version
# no result, expected since that version doesn't exist

yarn info nx@14.0.0 version --silent
14.0.0
```

```bash
npm view nx@^13.9.0 version
nx@13.9.0 '13.9.0'
nx@13.9.1 '13.9.1'
nx@13.9.2 '13.9.2'
nx@13.9.3 '13.9.3'
nx@13.9.4 '13.9.4'
nx@13.9.5 '13.9.5'
nx@13.9.6 '13.9.6'

yarn info nx@^13.9.0 version --silent
^13.9.0
```

When it comes to `pack`, the `npm pack` command will download a tarball for the specified package and version from the registry. On the other hand, `yarn pack` will create a tarball from the active workspace instead, it doesn't download it from the registry but rather creates a tarball of the files in the active workspace.

Reply:
I didn't add these two commands to this utility because there was no equivalent for Yarn and it felt wrong to add `npm` commands for `yarn`. IMHO, the intention feels clearer that's a fallback the way it was done, though it probably needed a comment describing why (still does, even if we keep it here).

Reply:
@leosvelperez gotcha. I understand your point, and in that case I'd like to undo those changes and create methods for `view` and `pack` that will not be part of PMC.